hphp/runtime/vm/jit/abi.h

```diff
@@ -78,25 +78,27 @@ PhysReg rret_type();
  */
 PhysReg rret(size_t i = 0);
 PhysReg rret_simd(size_t i);
-PhysReg rret_indirect();
 
 /*
  * Native argument registers.
  */
 PhysReg rarg(size_t i);
 PhysReg rarg_simd(size_t i);
+PhysReg rarg_ind_ret(size_t i);
 
 /*
  * Number of available argument registers.
  */
 size_t num_arg_regs();
 size_t num_arg_regs_simd();
+size_t num_arg_regs_ind_ret();
 
 /*
  * RegSet for a call with `n' arguments.
  */
 RegSet arg_regs(size_t n);
 RegSet arg_regs_simd(size_t n);
+RegSet arg_regs_ind_ret(size_t n);
```

Comment:
Huh, I don't either. I added it to alongside `arg_regs(size_t n)` and `arg_regs_simd(size_t n)` purely for balance since it's another `RegSet`. Looks like `arg_regs_simd(size_t n)` isn't used either.

What do you think I should do?

1. Remove `arg_regs_ind_ret(size_t n)`
2. Remove `arg_regs_ind_ret(size_t n)` and `arg_regs_simd(size_t n)`
3. Leave it as

Will leave as is with the next set of changes that I push for this PR, but will push another if you prefer 1 or 2.